test/e2e-olm/elasticsearch_test.go

```diff
@@ -320,22 +337,38 @@ func fullClusterRedeployTest(t *testing.T) {
 	//}
```

Comment:
can we remove the commented out code (above) since we now properly wait for restarts?